src/ast/CallOptions.ts

```diff
@@ -10,13 +10,13 @@ export type CallExpressionType = TaggedTemplateExpression | CallExpression | New
 export interface CallCreateOptions {
 	withNew: boolean;
 	args?: (ExpressionEntity | SpreadElement)[];
-	caller: CallExpressionType;
+	caller: Object;
```

Comment:
Is there any way to keep this as a more descriptive type? This analysis code can be quite complex to follow for users new to the codebase, so maintaining transparent types where possible would be useful in aiding understanding.

Reply:
Actually the only use of `caller` is to identify if the corresponding CallExpression has already been encountered in this run. Which does not exist in the case of implicit calls be builtins.
Would it be OK to instead rename `caller` to `callIdentifier` to make it more clear this is only an identification label?